src/main/java/frc/robot/subsystems/arm/ArmSubsystem.java

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.arm;
+
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.spark.SparkAbsoluteEncoder;
+import com.revrobotics.spark.SparkClosedLoopController;
+import com.revrobotics.spark.SparkMax;
+import com.revrobotics.spark.SparkRelativeEncoder;
+import com.revrobotics.spark.SparkBase.ControlType;
+import com.revrobotics.spark.SparkBase.PersistMode;
+import com.revrobotics.spark.SparkBase.ResetMode;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Configs.ArmConfigs;
+import frc.robot.Constants.ArmConstants;
+
+/**
+ * The ArmSubsystem class represents the robot's arm subsystem. It uses PIDF to
+ * pivot the arm.
+ */
+public class ArmSubsystem extends SubsystemBase {
+  private final SparkMax armSpark = new SparkMax(1, SparkMax.MotorType.kBrushless);
+  private final RelativeEncoder armEncoder = armSpark.getEncoder();
+  private final SparkClosedLoopController armClosedLoopController = armSpark.getClosedLoopController();
+
+  private double m_armAngularOffset = 0;
+
+  /**
+   * The constructor for the ArmSubsystem class configures the arm motor.
+   */
+  public ArmSubsystem() {
+
+    // configures arm motor
+    armSpark.configure(ArmConfigs.armConfig, ResetMode.kResetSafeParameters,
+        PersistMode.kPersistParameters);
+
+  }
+
+  @Override
+  public void periodic() {
+    double p = armEncoder.getPosition() * 360;
+    SmartDashboard.putNumber("position", p);
+    // System.out.println(p);
```

Comment:
Can you make sure that the setpoint and actual measured position is logged through advantageKit. Use `Intake/Setpoint/Position` and `Intake/Measured/Position` (and the velocity ones since manual control)